plugins/modules/cloud/scaleway/scaleway_container.py

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway Serverless container management module
+#
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_container
+short_description: Scaleway Container management module
+author: Guillaume MARTINEZ (@Lunik)
+description:
+  - This module manages container on Scaleway account
```

Comment:
```suggestion
  - This module manages container on Scaleway account.
```
Descriptions should always have complete sentences in each paragraph. (I'm not sure how the URL fits into this paragraph/sentence though, that's up for you to decide.)

Reply:
I have removed the URL and properly finished sentences with a `.`